src/cashweb/relay/constructors.ts

```diff
@@ -185,6 +186,26 @@ export class MessageConstructor {
     }
   }
 
+  constructForwardEntry({
+    from,
+    address,
+    content,
+  }: {
+    from: string
+    address: string
+    content: Array<MessageItem>
+  }) {
+    assert(typeof from === 'string', 'original sender name is wrong type')
+    assert(typeof address === 'string', 'original sender address is wrong type')
+    const forwardAsString = JSON.stringify({ from, address, content })
```

Comment:
So, harry did a few like this, but I would use an actual protobuf payload for this.